app/assets/stylesheets/helpers/_header.scss

```diff
@@ -516,6 +516,22 @@ $covid-grey: #262828;
   }
 }
 
+.global-bar-title__icon {
+  display: inline-block;
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-align: center;
+  align-items: center;
+  box-sizing: border-box;
+  padding-left: 50px;
+  min-height: 45px;
+  background-image: image-url("covid-arrow.png"); // PNG fallback for SVG
+  background-image: image-url("covid-arrow.svg");
+  background-repeat: no-repeat;
+  background-position: 0 50%;
+  background-size: 37px auto;
+}
```

Comment:
Focus state hides the arrow:

<img width="941" alt="Screenshot 2020-04-14 at 15 42 41" src="https://user-images.githubusercontent.com/861310/79237927-a4827b00-7e66-11ea-814a-a10501c1ce2d.png">

Reply:
We seem to have solved this problem for the same link on https://www.gov.uk/coronavirus. Suggest you copy the markup/styles from there.

Reply:
I've updated it to match.